scheduling-api-http/build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     // see scheduling/PropertyDecrypter.java for default settings
     compile group: 'com.github.ulisesbocchio', name: 'jasypt-spring-boot', version: '1.9'
 
-    runtime 'org.hsqldb:hsqldb'
+    runtime 'org.hsqldb:hsqldb:2.7.2:jdk8'
```

Comment:
A small question, can't we use the parent-bom for this project ?

Reply:
It uses parent-bom normally, not sure

Reply:
@mboussaa using the parent-bom means that you should not specifiy a version in parent-bom controlled projects.